### PR TITLE
1c_exporter role added for sarvar

### PR DESCRIPTION
**Pull request by rrozanoff**

### CODE REVIEW

File: host_vars/sarvar.yml [hunk 1]
```diff
@@ -44,3 +44,7 @@ scrape_config:
     static_configs:
       - targets:
           - 'localhost:9187'
+  - job_name: '1c_exporter'
+    static_configs:
+      - targets:
+         - 'localhost:{{ 1c_exporter_port }}
```
Comment (ivzyukov) [inline host_vars/sarvar.yml:50]: metrics_path ?

File: roles/1c_exporter/files/settings.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        - ragent
+        - rmngr
+      timerNotyfy: 10
+MSURL:         # Не обязательный параметр
+MSUSER:        # Не обязательный параметр
+MSPAS:         # Не обязательный параметр
+RAC:
+  Path: "/opt/1cv8/x86_64/8.3.19.1351/rac"
+  Port: "1545"      # Не обязательный параметр
+  Host: "sarvar" # Не обязательный параметр
```
Comment (ivzyukov) [inline roles/1c_exporter/files/settings.yaml:52]: use variable for it, or ansible fact

File: roles/1c_exporter/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: handlers
```
Comment (ivzyukov) [inline roles/1c_exporter/meta/main.yml:3]: add handler in roles/handlers/handlers/main.yml

File: roles/1c_exporter/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    name: 1c_exporter
+    user: root
+    image: rrozanoff/1c_exporter:{{ 1c_exporter_version }}
+    volumes:
+      - "{{ data_dir }}/1c_exporter/conf/settings.yaml:/settings.yaml:ro"
+    command:
+        "/1c_exporter \
+          -port={{ 1c_exporter_port }} \
+          --settings=/settings.yaml"
+
```
Comment (ivzyukov) [inline roles/1c_exporter/tasks/main.yml:27]: empty line

**Review by ivzyukov — CHANGES_REQUESTED** [review]

File: host_vars/sarvar.yml (hunk 1 above)
Comment (rrozanoff) [reply host_vars/sarvar.yml:51]: node_exporter does not need one, why for 1c_exporter?

Comment (ivzyukov) [reply host_vars/sarvar.yml:51]: image

File: host_vars/sarvar.yml [hunk 5]
```diff
@@ -5,6 +5,7 @@ prometheus_version: latest
 psql_exporter_version: v0.10.0
 my_timezone: Europe/Moscow
 my_domain: kettler
+1c_host: sarvar
```
Comment (ivzyukov) [inline host_vars/sarvar.yml:8]: we already have it, check {{ ansible_host }} fact

Comment (ivzyukov) [inline host_vars/sarvar.yml:8]: it should be 1c_host: {{ inventory_hostname }}

**Pull request merged**